install_config/configuring_azure.adoc

```diff
@@ -19,6 +19,12 @@ Azure disk as persistent storage] for application data. After Azure is
 configured properly, some additional configurations need to be completed on the
 {product-title} hosts.
 
+[[configuring-azure-permissions]]
+== Permissions
+Configuring Azure for {product-title} requires following roles:
+
+* *Contributor* - To create and manage all types of Azure resources.
```

Comment:
This are very generic permissions, do we have anything more security restrictive?